src/screens/Account/AccountActions.js

```diff
@@ -56,6 +54,12 @@ const AccountActions = ({
     [accountId, parentId, navigation],
   );
 
+  const ManageAction =
```

Comment:
this shouldn't have a uppercase first character if it's not a component.

Reply:
fixed the manageAction component actually returns null in the case of trc tokens however a component returning null isn't falsy and i can't check against it in order to apply the right layout here :disappointed: